.github/workflows/unit-tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         operating-system: [ubuntu-latest]
-        php-versions: ['7.2','7.3', '7.4']
+        php-versions: ['7.2', '7.4', '8.0']
```

Comment:
I think it's better to revert 6fc5136a46aba4b15426477f3017cacf0f81d497. We will update the minimum PHP version.